check-3-5/main.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+// ���ڸ� �Է� �޾Ƽ� 3�� ��� �Ǵ� 5�� ���� t �ƴϸ� f
+
+int main() {
+	int a;
+	scanf_s("%d", &a);
+
+	if ((a % 3 == 0))
```

Comment:
소괄호는 한번으로 줄이는게 좋을 것 같습니다!
그리고 들여쓰기를 할거라면 중괄호도 같이 쓰는게 좋을 것 같네요!
```suggestion
	if (a % 3 == 0) {
```